java/arcs/android/storage/database/DatabaseImpl.kt

```diff
@@ -211,19 +211,9 @@ class DatabaseImpl(
 
   /* Initializes the [PrimitiveType] values. */
   private fun initializePrimitiveTypes(db: SQLiteDatabase) {
```

Comment:
can you update the comment and rename this to something else eg initializeTypesTable